api/src/form_schema/forms/sf424a.py

```diff
@@ -831,6 +831,37 @@
             },
         }
     },
+    # Budget sections decomposition
+    # Transform row-oriented activity_line_items array to column-oriented arrays
+    # organized by section type (budget_summary, budget_categories, etc.)
+    "budget_sections": {
+        "xml_transform": {
+            "type": "conditional",
+            "target": "BudgetSections",
+            "conditional_transform": {
+                "type": "array_decomposition",
+                "source_array_field": "activity_line_items",
+                "field_mappings": {
+                    "BudgetSummaries": {
+                        "item_field": "budget_summary",
+                        "total_field": "total_budget_summary",
```

Comment:
To keep things a bit more generic, should we name these fields a bit different? `item_field` might be fine, but it could be that `total_field` is specific to this case.